server/src/main/java/org/apache/iotdb/db/engine/StorageEngine.java

```diff
@@ -346,4 +346,10 @@ public synchronized boolean deleteAll() {
     return true;
   }
 
+  public void rmStorageGroupInfo(String storageGroupName) {
```

Comment:
rename to deleteStorageGroup(), call deleteAllDataFilesInOneStorageGroup() first in this method.